### ISSUE TITLE
[Bug] Optifine Crashes When Making a New World

### ISSUE THREAD

**Issue by sdasdasdasdaasd** [sp614x/optifine#7582]
Optifine 1.20.1 has been crashing when I make a new world.

I can open and launch the game no problem. However, when I make a new world it'll complete to 100% show me the blank dirt background and promptly crash. This _only_ happens when launching with Optifine.

I cannot show you the error report because it refuses to show _me_ and just closes the launcher. It is an error 1 though. This _only_ happens when launching with Optifine.

I downloaded Optifine HD U I6 pre6 _and_ Optifine HD U I5, but both versions are having this issue.

I've tried restarting my computer. Reinstalling the launcher, reinstalling Optifine, reinstalling Java. I've deleted the config folder, I have no mods. I've tried downloading forge just to see it that would help (spoilers, it didn't), I've also tried Curse Forge. I've tried starting the world in the official 1.20.1 and then loading it with Optifine. I've set all my setting to the fastest option. Both fast math and smooth FPS are off. I've also tried making Optifine launch with the javawa file, but the launcher gets mad at me and won't launch Optifine at all. I would try to update my drivers but for a reason I can't remember my computer can't use NVIDIA. I would just delete the .minecraft file itself, but as stated the issue only arises when launching with Optifine, plus I feel redownloading the launcher covers this. 

I used the standalone version of Optifine to download.
Link here: https://optifine.net/downloadsl

Here is the latest log file.
[latest.log](https://github.com/sp614x/optifine/files/13540263/latest.log)

Please help, this has been driving me insane. Thank you in advance.

**Comment by Ari-RERA-43** [comment]
Can you try default video settings

**Comment by FewerElk** [comment]
I have forge

**Comment by sdasdasdasdaasd** [comment]
> Can you try default video settings
I have, it was my original guess as to why it was crashing.


> > I have the same issue, with solo worlds and servers : when I try to connect to a server I crash
> 
> Are you using either Forge or Fabric? If so, which?
I am not using either


> > I have forge for mc 1.20.1
> 
> What Forge version? Are you sure it's the correct version for your OptiFine version (ie. Forge 47.0.35 for OptiFine 1.20.1 I5)?
I am not using Forge.

**Comment by sdasdasdasdaasd** [comment]
I fixed it! I am free!

I tried downloading forge (47.0.35) again. It ran perfectly. Good. So then I added Optifine to my mods folder. It crashes. Not good. I then tried changing the executable again. It doesn't work. BUT, this time, I notice it says the java isn't compatible. A-HA! I download ver 17 of Java. It accepts that as the executable, but Optifine is _still_ crashing. I look to the mod folder and in a bid of desperation I think to extract the files. I did not think to do it before, because nothing on any site had told me to, and it wasn't a zip file. Oh how naive.
I run the game.
It loads the world.
I am free.
